tags.tf

```diff
@@ -24,7 +24,7 @@ locals {
     local.tags,
     var.runner_tags,
     # overwrites the `Name` key from `local.tags`
-    var.overrides["name_docker_machine_runners"] == "" ? { Name = format("%s-docker-machine", var.environment) } : { Name = var.overrides["name_docker_machine_runners"] },
+    var.overrides["name_docker_machine_runners"] == "" ? { Name = substr(format("%s-docker-machine", var.environment), 0, 28) } : { Name = var.overrides["name_docker_machine_runners"] },
```

Comment:
Sorry a bit late, made more or less Yesterday the sam fix. I think 20 was even too long, reduced it to 12 and removed docker-machine. Can you rebase the PR?

Reply:
Yeap, fixed.